libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -219,10 +220,7 @@ bool GLSLPostProcessor::process(const std::string& inputShader,
         case MaterialBuilder::Optimization::NONE:
             if (mSpirvOutput) {
                 SpvOptions options;
-#ifndef NDEBUG
-                options.generateDebugInfo = true;
-                options.disableOptimizer = true;
```

Comment:
Remove of this line is intentional (optimization and debugInfo are now orthogonal)